tests/ha/test_cluster_shutdown_start.py

```diff
@@ -626,6 +625,9 @@ def test_write_after_cluster_restart(self):
         LOGGER.info("Step 3: Cluster restarted fine and all Pods online.")
         LOGGER.info("STEP 4: Perform WRITEs with variable object sizes. 0B + (1KB - 512MB). "
                     "Verify READs and DI on the written data.")
+        users = self.mgnt_ops.create_account_users(nusers=1)
+        self.test_prefix = 'test-29470-1'
+        self.s3_clean.update(users)
```

Comment:
Should skip read also?

Reply:
No this is actually after cluster restarted, so read can be done.